inst/qml/Descriptives.qml

```diff
@@ -83,6 +83,7 @@ Form
 				enabled: plotVariables.checked || plotCorrelationMatrix.checked
 
 				indent:		true
+				CheckBox {			name: "overlay";			label: qsTr("Overlay")								}
```

Comment:
```suggestion
				CheckBox {			name: "distPlotOverlay";			label: qsTr("Overlay split levels in one plot"); enabled: splitBy.count > 0								}
```
1. change of the option name: It is good to make sure that the name of the option makes it clear which output it affects. In this case, the distribution plots.
2. change the label: be more descriptive to the user
3. enabled: enabling the option only if splitBy is not empty makes it clearer to the user what the button does.

Reply:
The question in general is whether this option should be presented here as is, as it affects only the distribution plots but not the correlation plots. The alternative option would be to implement this split for the correlation plots as well (which would be ideal, I think). But this is already a good step forward, so I would not be that opposed to let it be like this. We can revisit the correlation plots in the future.

Reply:
Also, it would be good to add description of this button in the help file (you can find the file inside `inst/help/Descriptives.md`).